docs/channel/README.md

```diff
@@ -20,13 +16,8 @@ The dataplane is designed in a way that it only communicates the control plane v
 
 ## Comparison with other KafkaChannel implementations
 
-You may find a comparison of this new KafkaChannel implementation with
-the [consolidated KafkaChannel](https://github.com/knative-extensions/eventing-kafka/blob/main/pkg/channel/consolidated/README.md)
-and
-the [distributed KafkaChannel](https://github.com/knative-extensions/eventing-kafka/blob/main/pkg/channel/distributed/README.md)
-implementations.
 
-|                         | New KafkaChannel                                        | Consolidated KafkaChannel                               | Distributed KafkaChannel                                                             |
+|                         | New KafkaChannel                                        | Consolidated KafkaChannel (Not supported anymore)                               | Distributed KafkaChannel (Not supported anymore)                                                             |
 |-------------------------|---------------------------------------------------------|---------------------------------------------------------|--------------------------------------------------------------------------------------|
```

Comment:
Though I made this changes and kept the comparison between channels, I am open for any suggestion. Thanks!

Reply:
I think that's reasonable! In case people need to refer back this section when they are transitioning.